src/app/config/config.ts

```diff
@@ -105,6 +104,7 @@ const awsConfig: AwsConfig = {
   ...basicVars.awsConfig,
   s3,
   virusScannerLambda,
+  // virusScannerQuarantineS3Bucket: 'dud',
```

Comment:
haha is this intended 😅

Reply:
oh OOPS I was simulating the virus scanner being missing 😓 good catch and thanks!

Reply:
Fixed in [6427556](https://github.com/opengovsg/FormSG/pull/6767/commits/64275561bac9962166b0bb381026f22219485a6d)